### PR TITLE
[LB] LTM new org and Tunnel offramp how-to guide

### PR DESCRIPTION
**Pull request by RebeccaTamachiro**
PCX-8256

### CODE REVIEW

File: content/load-balancing/local-traffic-management/_index.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+{{</Aside>}}
+
+---
+
+## Health monitor support
+
+Since traffic steering decisions or failover mechanisms rely on the health information of pools and origins, being able to input your virtual or private IPs directly as origins within your load balancer means you are able to better leverage existing health monitoring.
+
+Before, you could only enter tunnel addresses in your load balancer and configure health monitor requests to the tunnels themselves. Now, once the origin and VNet tunnel association is configured, Cloudflare determines not only the tunnel health but also the health of corresponding virtual or private IP targets.
```
Comment (marciocloudflare) [inline content/load-balancing/local-traffic-management/_index.md]: Do we explain what a VNet is? I think we should at least create a link to somewhere where this is explained, as folks might not know what a VNet is.

File: content/load-balancing/local-traffic-management/ltm-tunnels-setup.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+---
+
+# Set up private IPs with Cloudflare Tunnel
+
+Consider the following steps to learn how to configure Cloudflare local traffic management (LTM) solution, using [Cloudflare Tunnel](/cloudflare-one/connections/connect-networks/) as the off-ramp to securely connect to your private/internal services.
+
+## 1. Configure a Cloudflare tunnel with an assigned VNet
+
+The specific configuration steps can vary depending on your infrastructure and services you are looking to connect.
+If you are not familiar with Cloudflare Tunnel, the pages linked on each step provide more guidance.
```
Comment (marciocloudflare) [inline content/load-balancing/local-traffic-management/ltm-tunnels-setup.md]:
```suggestion

If you are not familiar with Cloudflare Tunnel, the pages linked on each step provide more guidance.
```

Comment (marciocloudflare) [reply content/load-balancing/local-traffic-management/ltm-tunnels-setup.md]: It seemed like you wanted this sentence to be on another paragraph. However, without a space between sentences, this one would be rendered as a part of the sentence on line 13, not a new paragraph.

File: content/load-balancing/local-traffic-management/ltm-tunnels-setup.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+pcx_content_type: how-to
+title: Set up private IPs with Tunnel
+weight: 2
+---
+
+# Set up private IPs with Cloudflare Tunnel
+
+Consider the following steps to learn how to configure Cloudflare local traffic management (LTM) solution, using [Cloudflare Tunnel](/cloudflare-one/connections/connect-networks/) as the off-ramp to securely connect to your private/internal services.
+
+## 1. Configure a Cloudflare tunnel with an assigned VNet
```
Comment (marciocloudflare) [inline content/load-balancing/local-traffic-management/ltm-tunnels-setup.md]: Consider replacing VNet with virtual network here as well

File: content/load-balancing/local-traffic-management/ltm-tunnels-setup.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+{{<Aside type="warning">}}
+
+All origins with private IPs must have `virtual_network_id` specified.
+
+{{</Aside>}}
+
+```bash
+$ curl --request PATCH \
+  --url https://api.cloudflare.com/client/v4/accounts/<account_id>/load_balancers/pools/<pool_id> \
```
Comment (marciocloudflare) [inline content/load-balancing/local-traffic-management/ltm-tunnels-setup.md]:
```suggestion
https://api.cloudflare.com/client/v4/accounts/<account_id>/load_balancers/pools/<pool_id> \
```

Comment (marciocloudflare) [reply content/load-balancing/local-traffic-management/ltm-tunnels-setup.md]: Looking at our guidelines for curl commands, I think you can skip the `--url` argument here

**Review by marciocloudflare — APPROVED** [review]

File: content/load-balancing/local-traffic-management/ltm-tunnels-setup.md (hunk 2 above)
Comment (RebeccaTamachiro) [reply content/load-balancing/local-traffic-management/ltm-tunnels-setup.md]: Oh, not sure what happened here/ why the new line. Good catch! I guess I actually preferred it as only one paragraph and then just ignored whatever was happening in the markdown. Have removed the new line so it's not confusing.

**Pull request merged**